apps/wallet-dashboard/components/Dialogs/Staking/timelock/UnstakeTimelockedObjectsDialog.tsx

```diff
@@ -0,0 +1,252 @@
+                    addNotification('Unstake transaction has been sent');
+                },
+                onError: () => {
+                    addNotification('Unstake transaction was not sent', NotificationType.Error);
+                },
+            },
+        );
+    }
+
+    function handleCopySuccess() {
+        addNotification('Copied to clipboard');
+    }
+
+    return (
+        <Dialog open onOpenChange={onClose}>
+            <Layout>
+                <Header title="Unstake" onClose={onClose} />
+                <LayoutBody>
+                    <div className="flex flex-col gap-md">
+                        <Validator
+                            address={groupedTimelockedObjects.validatorAddress}
+                            isSelected
+                            showActiveStatus
+                        />
+
+                        <TimelockStakeObjectsOverview
+                            rewardsPool={validatorInfo?.rewardsPool}
+                            groupedTimelockedObjects={groupedTimelockedObjects}
+                            handleCopySuccess={handleCopySuccess}
+                        />
+                    </div>
+                </LayoutBody>
+                <LayoutFooter>
+                    <Button
+                        onClick={handleTimelockedUnstake}
+                        text="Unstake"
+                        icon={
+                            isTransactionPending || isUnstakeTxLoading ? (
+                                <LoadingIndicator />
+                            ) : undefined
+                        }
+                        disabled={
+                            !timelockedUnstakeTx || isTransactionPending || isUnstakeTxLoading
+                        }
+                        type={ButtonType.Secondary}
+                        fullWidth
+                    />
+                </LayoutFooter>
+            </Layout>
+        </Dialog>
+    );
+}
+
+interface TimelockStakeObjectsOverviewProps {
+    groupedTimelockedObjects: TimelockedStakedObjectsGrouped;
+    handleCopySuccess: () => void;
+    rewardsPool: string | undefined;
+}
+
+function TimelockStakeObjectsOverview({
+    groupedTimelockedObjects: { stakes, validatorAddress, stakeRequestEpoch },
+    handleCopySuccess,
+    rewardsPool,
+}: TimelockStakeObjectsOverviewProps) {
+    const stakeId = stakes[0].timelockedStakedIotaId;
+
+    return (
+        <>
+            {stakeId && (
+                <ValidatorStakingData
+                    key={stakeId}
+                    validatorAddress={validatorAddress}
+                    stakeId={stakeId}
+                    isUnstake
+                />
+            )}
+
+            <Panel hasBorder>
+                <div className="flex flex-col gap-y-sm p-md">
+                    <KeyValueInfo
+                        keyText="Stake Request Epoch"
+                        value={stakeRequestEpoch}
+                        fullwidth
+                    />
```

Comment:
We could do like in unstake including a `Current Epoch Ends` and `Your stake`
- Current epoch ends
- Your stake (showing the sum of the amount of the all stakes)
- Rewards earned
- Total unstaked Timelocked (Your stake + Rewards earned)